run

```diff
@@ -84,35 +95,65 @@ else
     else
         measurement=`cat $MANIFEST_FILE | jq -r '.config.measurement.default'`
     fi
+
+    ## Determine measurement
+    # Check if autodetect, otherwise use what was read in from config.json/manifest.json
+    if [[ $measurement = 'auto-detect' ]]; then
+        echo "Auto-detecting input file measurement"
+        # Get measurement from config
+        measurement=`cat $CONFIG_FILE | jq '.inputs.nifti.object.measurements[0]'`
```

Comment:
I believe you can use `jq -r` and you would not need the next two lines.

Reply:
Implemented this change.